src/gh/diffCheck/diffCheck/df_visualization.py

```diff
@@ -13,18 +13,31 @@ class DFVizSettings:
     This class compiles the settings for the visualization into one object
     """
 
-    def __init__(self, valueType, upper_threshold, lower_threshold, palette, legend_height, legend_width, legend_plane, histogram_scale_factor):
+    def __init__(self
```

Comment:
missing "," after self. not sure why it worked in your case. on my computer the gh component was throwing an error